react/features/toolbox/components/MuteEveryoneButton.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import { createToolbarEvent, sendAnalytics } from '../../analytics';
+import { openDialog } from '../../base/dialog';
+import { translate } from '../../base/i18n';
+import { IconMicDisabled } from '../../base/icons';
+import { connect } from '../../base/redux';
+import { AbstractButton, type AbstractButtonProps } from '../../base/toolbox';
+import MuteEveryoneDialog from '../../remote-video-menu/components/web/MuteEveryoneDialog';
```

Comment:
Why not import directly from the feature?

Reply:
@hristoterezov Same answer as before.
@gimre Note that this button is not on web/ so it should be used on mobile, and then you are importing a web dialog, which is a no-no.

Reply:
@gimre Could you please either change the import to use the feature or either add a comment why this is problematic?

Reply:
Also please address @saghul comment. I guess you can move the component to the web folder since it is anyway imported only by the web toolbox.

Reply:
moved it to web/